contracts/src/v0.8/KeeperRegistry.sol

```diff
@@ -625,7 +643,16 @@ contract KeeperRegistry is
   function getMaxPaymentForGas(uint256 gasLimit) public view returns (uint96 maxPayment) {
     (uint256 gasWei, uint256 linkEth) = _getFeedData();
     uint256 adjustedGasWei = _adjustGasPrice(gasWei, false);
-    return _calculatePaymentAmount(gasLimit, adjustedGasWei, linkEth);
+    uint256 l1CostWei = 0;
+    bytes memory data = new bytes(0);
+    if (PAYMENT_MODEL == PaymentModel.OPTIMISM) {
+      data = bytes.concat(ESTIMATED_MSG_DATA, L1_FEE_DATA_PADDING);
+      l1CostWei = OPTIMISM_ORACLE.getL1Fee(data);
```

Comment:
nit. i think you can do away with another variable and just pass `bytes.concat(ESTIMATED_MSG_DATA, L1_FEE_DATA_PADDING)` here

Reply:
Done